APR-Core/Config.lua

```diff
@@ -47,6 +47,7 @@ function APR.settings:InitializeSettings()
             autoRepair = false,
             autoSkipCutScene = true,
             autoFlight = true,
+            autoShareQuestWithFriend = false,
```

Comment:
### Idea
Adds anothervariable to false by default like `firstAutoShareQuestWithFriend`.
and in the "QUEST_ACCEPTED" add a popup if CheckIfPartyMemberIsFriend == true to ask if it wants to enbale the autoShareQuestWithFriend option which is disabled by default

Since it's a new feature so people will know about it

*look QuestionPopUp*